package.json

```diff
@@ -7,6 +7,7 @@
     "npm": "~3.8.x"
   },
   "dependencies": {
+    "app": "^0.1.0",
```

Comment:
I have a feeling this is codeclimate being tricked by the Rails directory format....
[app](https://www.npmjs.com/package/app) looks like a web development framework and we're definitely not using that